2025-07-04 12:18:38,025 - ERROR - 
❌ Errore nella verifica della tabella Oracle: DPI-1047: Cannot locate a 64-bit Oracle Client library: "The specified module could not be found". See https://cx-oracle.readthedocs.io/en/latest/user_guide/installation.html for help
2025-07-04 12:25:31,399 - ERROR - 
❌ Errore nella verifica della tabella Oracle: DPI-1047: Cannot locate a 64-bit Oracle Client library: "The specified module could not be found". See https://cx-oracle.readthedocs.io/en/latest/user_guide/installation.html for help
2025-07-04 14:44:04,821 - ERROR - 
❌ Errore nella verifica della tabella Oracle: DPI-1047: Cannot locate a 64-bit Oracle Client library: "The specified module could not be found". See https://cx-oracle.readthedocs.io/en/latest/user_guide/installation.html for help
2025-07-04 15:00:14,052 - ERROR - 
❌ Errore nella verifica della tabella Oracle: DPI-1047: Cannot locate a 64-bit Oracle Client library: "The specified module could not be found". See https://cx-oracle.readthedocs.io/en/latest/user_guide/installation.html for help
2025-07-04 15:05:56,649 - ERROR - 
❌ Errore nella verifica della tabella Oracle: DPI-1047: Cannot locate a 64-bit Oracle Client library: "The specified module could not be found". See https://cx-oracle.readthedocs.io/en/latest/user_guide/installation.html for help
2025-07-04 15:07:04,686 - ERROR - 
❌ Errore nella verifica della tabella Oracle: ORA-03113: end-of-file on communication channel
Process ID: 0
Session ID: 0 Serial number: 0
Help: https://docs.oracle.com/error-help/db/ora-03113/
2025-07-04 15:10:30,077 - ERROR - 
❌ Errore nella verifica della tabella Oracle: ORA-03113: end-of-file on communication channel
Process ID: 0
Session ID: 0 Serial number: 0
Help: https://docs.oracle.com/error-help/db/ora-03113/
2025-07-04 15:17:34,055 - ERROR - 
❌ Errore nella verifica della tabella Oracle: ORA-03113: end-of-file on communication channel
Process ID: 0
Session ID: 0 Serial number: 0
Help: https://docs.oracle.com/error-help/db/ora-03113/
2025-07-04 15:18:18,411 - ERROR - 
❌ Errore nella verifica della tabella Oracle: ORA-03113: end-of-file on communication channel
Process ID: 0
Session ID: 0 Serial number: 0
Help: https://docs.oracle.com/error-help/db/ora-03113/
2025-07-04 15:20:32,234 - ERROR - 
❌ Errore nella verifica della tabella Oracle: ORA-03113: end-of-file on communication channel
Process ID: 0
Session ID: 0 Serial number: 0
Help: https://docs.oracle.com/error-help/db/ora-03113/
2025-07-04 15:25:28,210 - ERROR - 
❌ Errore nella verifica della tabella Oracle: ORA-03113: end-of-file on communication channel
Process ID: 0
Session ID: 0 Serial number: 0
Help: https://docs.oracle.com/error-help/db/ora-03113/
2025-07-04 15:29:53,548 - ERROR - 
❌ Errore nella verifica della tabella Oracle: ORA-03113: end-of-file on communication channel
Process ID: 0
Session ID: 0 Serial number: 0
Help: https://docs.oracle.com/error-help/db/ora-03113/
2025-07-04 15:41:35,732 - ERROR - 
❌ Errore nella verifica della tabella Oracle: DPI-1047: Cannot locate a 64-bit Oracle Client library: "The specified module could not be found". See https://cx-oracle.readthedocs.io/en/latest/user_guide/installation.html for help
2025-07-04 15:47:47,177 - ERROR - 
❌ Errore nella verifica della tabella Oracle: DPI-1047: Cannot locate a 64-bit Oracle Client library: "The specified module could not be found". See https://cx-oracle.readthedocs.io/en/latest/user_guide/installation.html for help
2025-07-04 16:05:04,999 - ERROR - 
❌ Errore nella verifica della tabella Oracle: DPI-1047: Cannot locate a 64-bit Oracle Client library: "The specified module could not be found". See https://cx-oracle.readthedocs.io/en/latest/user_guide/installation.html for help
2025-07-04 16:06:11,803 - ERROR - 
❌ Errore nella verifica della tabella Oracle: ORA-03113: end-of-file on communication channel
Process ID: 0
Session ID: 0 Serial number: 0
Help: https://docs.oracle.com/error-help/db/ora-03113/
2025-07-04 16:30:10,211 - ERROR - 
❌ Errore nella verifica della tabella Oracle: ORA-03113: end-of-file on communication channel
Process ID: 0
Session ID: 0 Serial number: 0
Help: https://docs.oracle.com/error-help/db/ora-03113/
2025-07-04 16:31:33,712 - ERROR - 
❌ Errore nella verifica della tabella Oracle: ORA-03113: end-of-file on communication channel
Process ID: 0
Session ID: 0 Serial number: 0
Help: https://docs.oracle.com/error-help/db/ora-03113/
2025-07-04 16:37:53,105 - ERROR - 
❌ Errore nella verifica della tabella Oracle: DPI-1047: Cannot locate a 64-bit Oracle Client library: "The specified module could not be found". See https://cx-oracle.readthedocs.io/en/latest/user_guide/installation.html for help
2025-07-04 16:39:37,117 - ERROR - 
❌ Errore nella verifica della tabella Oracle: DPI-1047: Cannot locate a 64-bit Oracle Client library: "The specified module could not be found". See https://cx-oracle.readthedocs.io/en/latest/user_guide/installation.html for help
2025-07-04 16:48:18,706 - ERROR - 
❌ Errore nella verifica della tabella Oracle: ORA-03113: end-of-file on communication channel
Process ID: 0
Session ID: 0 Serial number: 0
2025-07-04 17:06:17,780 - ERROR - 
❌ Errore nella verifica della tabella Oracle: ORA-03113: end-of-file on communication channel
Process ID: 0
Session ID: 0 Serial number: 0
2025-07-04 17:07:50,263 - ERROR - 
❌ Errore nella verifica della tabella Oracle: ORA-03113: end-of-file on communication channel
Process ID: 0
Session ID: 0 Serial number: 0
2025-07-04 17:09:37,838 - ERROR - 
❌ Errore nella verifica della tabella Oracle: connect() got an unexpected keyword argument 'encoding'
2025-07-04 17:11:19,779 - ERROR - 
❌ Errore nella verifica della tabella Oracle: DPY-6005: cannot connect to database (CONNECTION_ID=kO9/aZY9/KZAo9twJ9I6Ag==).
DPY-3010: connections to this database server version are not supported by python-oracledb in thin mode
Help: https://python-oracledb.readthedocs.io/en/latest/user_guide/troubleshooting.html#dpy-3010
2025-07-04 17:46:56,451 - ERROR - 
❌ Errore nella lettura dei codici a barre da Oracle: connect() got an unexpected keyword argument 'encoding'
2025-07-04 17:46:56,452 - ERROR - Il programma non può continuare senza i codici a barre.
2025-07-04 17:51:46,881 - ERROR - 
❌ Errore nella lettura dei codici a barre da Oracle: connect() got an unexpected keyword argument 'encoding'
2025-07-04 17:51:46,882 - ERROR - Il programma non può continuare senza i codici a barre.
2025-07-04 17:52:06,172 - ERROR - 
❌ Errore nella lettura dei codici a barre da Oracle: connect() got an unexpected keyword argument 'encoding'
2025-07-04 17:52:06,173 - ERROR - Il programma non può continuare senza i codici a barre.
2025-07-04 17:53:59,828 - ERROR - 
❌ Errore nell'elaborazione del file C:\Users\ansel\Work\projects\python\Email_Analyzer_v1\Mail Fittizie\test1.eml: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, head to: https://ai.google.dev/gemini-api/docs/rate-limits. [violations {
  quota_metric: "generativelanguage.googleapis.com/generate_content_free_tier_requests"
  quota_id: "GenerateRequestsPerMinutePerProjectPerModel-FreeTier"
  quota_dimensions {
    key: "model"
    value: "gemini-2.0-flash"
  }
  quota_dimensions {
    key: "location"
    value: "global"
  }
  quota_value: 15
}
, links {
  description: "Learn more about Gemini API quotas"
  url: "https://ai.google.dev/gemini-api/docs/rate-limits"
}
, retry_delay {
}
]
2025-07-04 17:53:59,835 - ERROR - Traceback (most recent call last):
  File "C:\Users\ansel\Work\projects\python\Email_Analyzer_v1\components\processing.py", line 25, in elabora_file
    risposta = chiedi_al_modello(prompt)
  File "C:\Users\ansel\Work\projects\python\Email_Analyzer_v1\components\ai_utils.py", line 112, in chiedi_al_modello
    response = model.generate_content(prompt)
  File "C:\Users\ansel\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\generativeai\generative_models.py", line 331, in generate_content
    response = self._client.generate_content(
        request,
        **request_options,
    )
  File "C:\Users\ansel\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\ai\generativelanguage_v1beta\services\generative_service\client.py", line 835, in generate_content
    response = rpc(
        request,
    ...<2 lines>...
        metadata=metadata,
    )
  File "C:\Users\ansel\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\gapic_v1\method.py", line 131, in __call__
    return wrapped_func(*args, **kwargs)
  File "C:\Users\ansel\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_unary.py", line 294, in retry_wrapped_func
    return retry_target(
        target,
    ...<3 lines>...
        on_error=on_error,
    )
  File "C:\Users\ansel\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_unary.py", line 156, in retry_target
    next_sleep = _retry_error_helper(
        exc,
    ...<6 lines>...
        timeout,
    )
  File "C:\Users\ansel\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_base.py", line 214, in _retry_error_helper
    raise final_exc from source_exc
  File "C:\Users\ansel\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_unary.py", line 147, in retry_target
    result = target()
  File "C:\Users\ansel\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\timeout.py", line 130, in func_with_timeout
    return func(*args, **kwargs)
  File "C:\Users\ansel\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\grpc_helpers.py", line 78, in error_remapped_callable
    raise exceptions.from_grpc_error(exc) from exc
google.api_core.exceptions.ResourceExhausted: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, head to: https://ai.google.dev/gemini-api/docs/rate-limits. [violations {
  quota_metric: "generativelanguage.googleapis.com/generate_content_free_tier_requests"
  quota_id: "GenerateRequestsPerMinutePerProjectPerModel-FreeTier"
  quota_dimensions {
    key: "model"
    value: "gemini-2.0-flash"
  }
  quota_dimensions {
    key: "location"
    value: "global"
  }
  quota_value: 15
}
, links {
  description: "Learn more about Gemini API quotas"
  url: "https://ai.google.dev/gemini-api/docs/rate-limits"
}
, retry_delay {
}
]

2025-07-04 17:53:59,836 - ERROR - 
❌ Errore nell'elaborazione del file: test1.eml
2025-07-07 11:26:43,214 - ERROR - 
❌ Errore nell'elaborazione del file C:\PYTHON\Email_Analyzer_v1.1\Mail Fittizie\test1.eml: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, head to: https://ai.google.dev/gemini-api/docs/rate-limits. [violations {
  quota_metric: "generativelanguage.googleapis.com/generate_content_free_tier_requests"
  quota_id: "GenerateRequestsPerMinutePerProjectPerModel-FreeTier"
  quota_dimensions {
    key: "model"
    value: "gemini-2.0-flash"
  }
  quota_dimensions {
    key: "location"
    value: "global"
  }
  quota_value: 15
}
, links {
  description: "Learn more about Gemini API quotas"
  url: "https://ai.google.dev/gemini-api/docs/rate-limits"
}
, retry_delay {
  seconds: 14
}
]
2025-07-07 11:26:43,316 - ERROR - Traceback (most recent call last):
  File "C:\PYTHON\Email_Analyzer_v1.1\components\processing.py", line 25, in elabora_file
    risposta = chiedi_al_modello(prompt)
  File "C:\PYTHON\Email_Analyzer_v1.1\components\ai_utils.py", line 94, in chiedi_al_modello
    response = model.generate_content(prompt)
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\generativeai\generative_models.py", line 331, in generate_content
    response = self._client.generate_content(
        request,
        **request_options,
    )
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\ai\generativelanguage_v1beta\services\generative_service\client.py", line 835, in generate_content
    response = rpc(
        request,
    ...<2 lines>...
        metadata=metadata,
    )
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\gapic_v1\method.py", line 131, in __call__
    return wrapped_func(*args, **kwargs)
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_unary.py", line 294, in retry_wrapped_func
    return retry_target(
        target,
    ...<3 lines>...
        on_error=on_error,
    )
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_unary.py", line 156, in retry_target
    next_sleep = _retry_error_helper(
        exc,
    ...<6 lines>...
        timeout,
    )
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_base.py", line 214, in _retry_error_helper
    raise final_exc from source_exc
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_unary.py", line 147, in retry_target
    result = target()
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\timeout.py", line 130, in func_with_timeout
    return func(*args, **kwargs)
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\grpc_helpers.py", line 78, in error_remapped_callable
    raise exceptions.from_grpc_error(exc) from exc
google.api_core.exceptions.ResourceExhausted: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, head to: https://ai.google.dev/gemini-api/docs/rate-limits. [violations {
  quota_metric: "generativelanguage.googleapis.com/generate_content_free_tier_requests"
  quota_id: "GenerateRequestsPerMinutePerProjectPerModel-FreeTier"
  quota_dimensions {
    key: "model"
    value: "gemini-2.0-flash"
  }
  quota_dimensions {
    key: "location"
    value: "global"
  }
  quota_value: 15
}
, links {
  description: "Learn more about Gemini API quotas"
  url: "https://ai.google.dev/gemini-api/docs/rate-limits"
}
, retry_delay {
  seconds: 14
}
]

2025-07-07 11:26:43,317 - ERROR - 
❌ Errore nell'elaborazione del file: test1.eml
2025-07-07 11:45:39,698 - ERROR - 
❌ Errore nell'elaborazione del file C:\PYTHON\Email_Analyzer_v1.1\Mail Fittizie\test1.eml: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, head to: https://ai.google.dev/gemini-api/docs/rate-limits. [violations {
  quota_metric: "generativelanguage.googleapis.com/generate_content_free_tier_requests"
  quota_id: "GenerateRequestsPerMinutePerProjectPerModel-FreeTier"
  quota_dimensions {
    key: "model"
    value: "gemini-2.0-flash"
  }
  quota_dimensions {
    key: "location"
    value: "global"
  }
  quota_value: 15
}
, links {
  description: "Learn more about Gemini API quotas"
  url: "https://ai.google.dev/gemini-api/docs/rate-limits"
}
, retry_delay {
  seconds: 18
}
]
2025-07-07 11:45:39,705 - ERROR - Traceback (most recent call last):
  File "C:\PYTHON\Email_Analyzer_v1.1\components\processing.py", line 25, in elabora_file
    risposta = chiedi_al_modello(prompt)
  File "C:\PYTHON\Email_Analyzer_v1.1\components\ai_utils.py", line 94, in chiedi_al_modello
    response = model.generate_content(prompt)
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\generativeai\generative_models.py", line 331, in generate_content
    response = self._client.generate_content(
        request,
        **request_options,
    )
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\ai\generativelanguage_v1beta\services\generative_service\client.py", line 835, in generate_content
    response = rpc(
        request,
    ...<2 lines>...
        metadata=metadata,
    )
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\gapic_v1\method.py", line 131, in __call__
    return wrapped_func(*args, **kwargs)
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_unary.py", line 294, in retry_wrapped_func
    return retry_target(
        target,
    ...<3 lines>...
        on_error=on_error,
    )
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_unary.py", line 156, in retry_target
    next_sleep = _retry_error_helper(
        exc,
    ...<6 lines>...
        timeout,
    )
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_base.py", line 214, in _retry_error_helper
    raise final_exc from source_exc
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_unary.py", line 147, in retry_target
    result = target()
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\timeout.py", line 130, in func_with_timeout
    return func(*args, **kwargs)
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\grpc_helpers.py", line 78, in error_remapped_callable
    raise exceptions.from_grpc_error(exc) from exc
google.api_core.exceptions.ResourceExhausted: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, head to: https://ai.google.dev/gemini-api/docs/rate-limits. [violations {
  quota_metric: "generativelanguage.googleapis.com/generate_content_free_tier_requests"
  quota_id: "GenerateRequestsPerMinutePerProjectPerModel-FreeTier"
  quota_dimensions {
    key: "model"
    value: "gemini-2.0-flash"
  }
  quota_dimensions {
    key: "location"
    value: "global"
  }
  quota_value: 15
}
, links {
  description: "Learn more about Gemini API quotas"
  url: "https://ai.google.dev/gemini-api/docs/rate-limits"
}
, retry_delay {
  seconds: 18
}
]

2025-07-07 11:45:39,705 - ERROR - 
❌ Errore nell'elaborazione del file: test1.eml
2025-07-07 11:49:16,527 - ERROR - 
❌ Errore nell'elaborazione del file C:\PYTHON\Email_Analyzer_v1.1\Mail Fittizie\test1.eml: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, head to: https://ai.google.dev/gemini-api/docs/rate-limits. [violations {
  quota_metric: "generativelanguage.googleapis.com/generate_content_free_tier_requests"
  quota_id: "GenerateRequestsPerMinutePerProjectPerModel-FreeTier"
  quota_dimensions {
    key: "model"
    value: "gemini-2.0-flash"
  }
  quota_dimensions {
    key: "location"
    value: "global"
  }
  quota_value: 15
}
, links {
  description: "Learn more about Gemini API quotas"
  url: "https://ai.google.dev/gemini-api/docs/rate-limits"
}
, retry_delay {
  seconds: 41
}
]
2025-07-07 11:49:16,551 - ERROR - Traceback (most recent call last):
  File "C:\PYTHON\Email_Analyzer_v1.1\components\processing.py", line 25, in elabora_file
    risposta = chiedi_al_modello(prompt)
  File "C:\PYTHON\Email_Analyzer_v1.1\components\ai_utils.py", line 94, in chiedi_al_modello
    response = model.generate_content(prompt)
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\generativeai\generative_models.py", line 331, in generate_content
    response = self._client.generate_content(
        request,
        **request_options,
    )
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\ai\generativelanguage_v1beta\services\generative_service\client.py", line 835, in generate_content
    response = rpc(
        request,
    ...<2 lines>...
        metadata=metadata,
    )
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\gapic_v1\method.py", line 131, in __call__
    return wrapped_func(*args, **kwargs)
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_unary.py", line 294, in retry_wrapped_func
    return retry_target(
        target,
    ...<3 lines>...
        on_error=on_error,
    )
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_unary.py", line 156, in retry_target
    next_sleep = _retry_error_helper(
        exc,
    ...<6 lines>...
        timeout,
    )
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_base.py", line 214, in _retry_error_helper
    raise final_exc from source_exc
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_unary.py", line 147, in retry_target
    result = target()
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\timeout.py", line 130, in func_with_timeout
    return func(*args, **kwargs)
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\grpc_helpers.py", line 78, in error_remapped_callable
    raise exceptions.from_grpc_error(exc) from exc
google.api_core.exceptions.ResourceExhausted: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, head to: https://ai.google.dev/gemini-api/docs/rate-limits. [violations {
  quota_metric: "generativelanguage.googleapis.com/generate_content_free_tier_requests"
  quota_id: "GenerateRequestsPerMinutePerProjectPerModel-FreeTier"
  quota_dimensions {
    key: "model"
    value: "gemini-2.0-flash"
  }
  quota_dimensions {
    key: "location"
    value: "global"
  }
  quota_value: 15
}
, links {
  description: "Learn more about Gemini API quotas"
  url: "https://ai.google.dev/gemini-api/docs/rate-limits"
}
, retry_delay {
  seconds: 41
}
]

2025-07-07 11:49:16,553 - ERROR - 
❌ Errore nell'elaborazione del file: test1.eml
2025-07-07 12:04:45,332 - ERROR - 
❌ Errore nell'elaborazione del file C:\PYTHON\Email_Analyzer_v1.1\Mail Fittizie\test1.eml: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, head to: https://ai.google.dev/gemini-api/docs/rate-limits. [violations {
  quota_metric: "generativelanguage.googleapis.com/generate_content_free_tier_requests"
  quota_id: "GenerateRequestsPerMinutePerProjectPerModel-FreeTier"
  quota_dimensions {
    key: "model"
    value: "gemini-2.0-flash"
  }
  quota_dimensions {
    key: "location"
    value: "global"
  }
  quota_value: 15
}
, links {
  description: "Learn more about Gemini API quotas"
  url: "https://ai.google.dev/gemini-api/docs/rate-limits"
}
, retry_delay {
  seconds: 12
}
]
2025-07-07 12:04:45,341 - ERROR - Traceback (most recent call last):
  File "C:\PYTHON\Email_Analyzer_v1.1\components\processing.py", line 25, in elabora_file
    risposta = chiedi_al_modello(prompt)
  File "C:\PYTHON\Email_Analyzer_v1.1\components\ai_utils.py", line 94, in chiedi_al_modello
    response = model.generate_content(prompt)
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\generativeai\generative_models.py", line 331, in generate_content
    response = self._client.generate_content(
        request,
        **request_options,
    )
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\ai\generativelanguage_v1beta\services\generative_service\client.py", line 835, in generate_content
    response = rpc(
        request,
    ...<2 lines>...
        metadata=metadata,
    )
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\gapic_v1\method.py", line 131, in __call__
    return wrapped_func(*args, **kwargs)
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_unary.py", line 294, in retry_wrapped_func
    return retry_target(
        target,
    ...<3 lines>...
        on_error=on_error,
    )
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_unary.py", line 156, in retry_target
    next_sleep = _retry_error_helper(
        exc,
    ...<6 lines>...
        timeout,
    )
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_base.py", line 214, in _retry_error_helper
    raise final_exc from source_exc
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\retry\retry_unary.py", line 147, in retry_target
    result = target()
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\timeout.py", line 130, in func_with_timeout
    return func(*args, **kwargs)
  File "C:\Users\aless\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\api_core\grpc_helpers.py", line 78, in error_remapped_callable
    raise exceptions.from_grpc_error(exc) from exc
google.api_core.exceptions.ResourceExhausted: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, head to: https://ai.google.dev/gemini-api/docs/rate-limits. [violations {
  quota_metric: "generativelanguage.googleapis.com/generate_content_free_tier_requests"
  quota_id: "GenerateRequestsPerMinutePerProjectPerModel-FreeTier"
  quota_dimensions {
    key: "model"
    value: "gemini-2.0-flash"
  }
  quota_dimensions {
    key: "location"
    value: "global"
  }
  quota_value: 15
}
, links {
  description: "Learn more about Gemini API quotas"
  url: "https://ai.google.dev/gemini-api/docs/rate-limits"
}
, retry_delay {
  seconds: 12
}
]

2025-07-07 12:04:45,341 - ERROR - 
❌ Errore nell'elaborazione del file: test1.eml
2025-07-09 10:41:41,484 - ERROR - 
❌ Il file dei codici a barre non esiste! (C:\PYTHON\Email_Analyzer_v1.1.1\Email_Analyzer_v1.1.1\codici_barre.csv)
2025-07-09 10:41:41,487 - ERROR - Il programma non può continuare senza il file dei codici a barre.
2025-07-10 12:35:22,370 - ERROR - 
❌ Il percorso specificato non è una cartella valida!
2025-07-10 12:35:52,203 - ERROR - 
❌ Errore nell'inserimento dati in Oracle: ORA-01401: inserted value too large for column
2025-07-10 13:02:21,552 - ERROR - 
❌ Errore nell'inserimento dati in Oracle: ORA-01401: inserted value too large for column
2025-07-10 13:08:07,250 - ERROR - 
❌ Errore nell'inserimento dati in Oracle: ORA-01401: inserted value too large for column
2025-07-10 14:28:23,362 - ERROR - 
❌ Errore nell'inserimento dati in Oracle: ORA-12899: valore troppo grande per la colonna "EATINERARI"."ORD_EATIN"."CODICE_FISCALE" (corrente: 181, massimo: 30)
2025-07-10 14:52:39,628 - ERROR - 
❌ Errore nell'inserimento dati in Oracle: ORA-12899: valore troppo grande per la colonna "EATINERARI"."ORD_EATIN"."CODICE_FISCALE" (corrente: 181, massimo: 30)
2025-07-10 14:57:54,172 - ERROR - 
❌ Errore nell'inserimento dati in Oracle: ORA-12899: valore troppo grande per la colonna "EATINERARI"."ORD_EATIN"."CODICE_FISCALE" (corrente: 181, massimo: 30)
2025-07-10 15:20:42,808 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-12899: valore troppo grande per la colonna "EATINERARI"."ORD_EATIN"."CODICE_FISCALE" (corrente: 181, massimo: 30)
2025-07-10 15:29:03,986 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-12899: valore troppo grande per la colonna "EATINERARI"."ORD_EATIN"."CODICE_FISCALE" (corrente: 181, massimo: 30)
2025-07-10 15:33:12,214 - ERROR - 
❌ Errore generale nell'inserimento dati in Oracle: name 'os' is not defined
2025-07-10 15:37:36,899 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-12899: valore troppo grande per la colonna "EATINERARI"."ORD_EATIN"."CODICE_FISCALE" (corrente: 181, massimo: 30)
2025-07-10 15:40:38,684 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-12899: valore troppo grande per la colonna "EATINERARI"."ORD_EATIN"."CODICE_FISCALE" (corrente: 181, massimo: 30)
2025-07-10 15:44:49,487 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-12899: valore troppo grande per la colonna "EATINERARI"."ORD_EATIN"."CODICE_FISCALE" (corrente: 181, massimo: 30)
2025-07-10 15:47:59,894 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-12899: valore troppo grande per la colonna "EATINERARI"."ORD_EATIN"."CODICE_FISCALE" (corrente: 181, massimo: 30)
2025-07-10 15:51:43,528 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-12899: valore troppo grande per la colonna "EATINERARI"."ORD_EATIN"."CODICE_FISCALE" (corrente: 181, massimo: 30)
2025-07-10 15:59:14,366 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-12899: valore troppo grande per la colonna "EATINERARI"."ORD_EATIN"."CODICE_FISCALE" (corrente: 181, massimo: 30)
2025-07-10 17:09:24,629 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-12899: valore troppo grande per la colonna "EATINERARI"."ORD_EATIN"."CODICE_FISCALE" (corrente: 181, massimo: 30)
2025-07-11 13:02:23,282 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-12899: valore troppo grande per la colonna "EATINERARI"."ORD_EATIN"."CODICE_FISCALE" (corrente: 181, massimo: 30)
2025-07-15 15:01:01,276 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail '20250710_103250_11_Inserimento nuova richiesta Eatinerari - Pacchetti di esperienze targati Eataly.eml': ORA-01401: inserted value too large for column
2025-07-15 15:02:32,474 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-15 15:06:35,474 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-16 09:14:19,644 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-16 09:45:26,956 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-16 09:50:08,725 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-16 09:53:41,695 - ERROR - 
❌ File con errore logico (doppio o altro): Inserimento nuova richiesta Eatinerari - A tavola con Eataly.msg
2025-07-16 09:53:44,259 - ERROR - 
❌ File con errore logico (doppio o altro): Inserimento nuova richiesta Eatinerari - Pacchetti di esperienze targati Eataly.msg
2025-07-16 09:53:46,587 - ERROR - 
❌ File con errore logico (doppio o altro): Notifica ordine 00167525.msg
2025-07-16 09:53:48,876 - ERROR - 
❌ File con errore logico (doppio o altro): Notifica ordine 00168844.msg
2025-07-16 09:53:51,454 - ERROR - 
❌ File con errore logico (doppio o altro): Notifica ordine 00194151.msg
2025-07-16 09:53:53,816 - ERROR - 
❌ File con errore logico (doppio o altro): PORTALE WELFARE PELLEGRINI CONFERMA INSERIMENTO RICHIESTA.msg
2025-07-16 09:53:56,081 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto ALESSIA MARIOTTI # SKY25043001590064.msg
2025-07-16 09:53:58,398 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto ALESSIO ARRIGO # SKY25052401704582.msg
2025-07-16 09:54:01,361 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto ALESSIO ARRIGO # SKY25052701720736.msg
2025-07-16 09:54:03,909 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto DOROTHY MAFFIA # SKY25061901874877.msg
2025-07-16 09:54:06,619 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto Giacomo Cavallo # SKY25050701621176.msg
2025-07-16 09:54:09,225 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto GIOVANNI GIORDANI # SKY25061101818808.msg
2025-07-16 09:54:13,710 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-16 09:54:16,213 - ERROR - 
❌ File con errore logico (doppio o altro): testdebub.eml
2025-07-16 11:02:40,198 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-16 11:04:41,698 - ERROR - 
❌ File con errore logico (doppio o altro): DEBUG - VOUCHER WILLIS WELFARE - ORDINE N.2250481.msg
2025-07-16 11:04:44,321 - ERROR - 
❌ File con errore logico (doppio o altro): Inserimento nuova richiesta Eatinerari - A tavola con Eataly.msg
2025-07-16 11:04:46,754 - ERROR - 
❌ File con errore logico (doppio o altro): Inserimento nuova richiesta Eatinerari - Pacchetti di esperienze targati Eataly.msg
2025-07-16 11:04:49,398 - ERROR - 
❌ File con errore logico (doppio o altro): Notifica ordine 00167525.msg
2025-07-16 11:04:54,600 - ERROR - 
❌ File con errore logico (doppio o altro): Notifica ordine 00194151.msg
2025-07-16 11:04:56,832 - ERROR - 
❌ File con errore logico (doppio o altro): PORTALE WELFARE PELLEGRINI CONFERMA INSERIMENTO RICHIESTA.msg
2025-07-16 11:04:59,640 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto ALESSIA MARIOTTI # SKY25043001590064.msg
2025-07-16 11:05:01,913 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto ALESSIO ARRIGO # SKY25052401704582.msg
2025-07-16 11:05:04,313 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto ALESSIO ARRIGO # SKY25052701720736.msg
2025-07-16 11:05:06,595 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto DOROTHY MAFFIA # SKY25061901874877.msg
2025-07-16 11:05:08,944 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto Giacomo Cavallo # SKY25050701621176.msg
2025-07-16 11:05:11,450 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto GIOVANNI GIORDANI # SKY25061101818808.msg
2025-07-16 11:05:14,961 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-16 11:10:08,075 - ERROR - 
❌ File con errore logico (doppio o altro): DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg
2025-07-16 11:24:59,528 - ERROR - 
❌ File con errore logico (doppio o altro): DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg
2025-07-16 11:42:23,811 - ERROR - 
❌ File con errore logico (doppio o altro): DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg
2025-07-16 14:13:09,042 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail '20250710_103250_11_Inserimento nuova richiesta Eatinerari - Pacchetti di esperienze targati Eataly.eml': ORA-01401: inserted value too large for column
2025-07-16 14:13:53,004 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-16 14:15:57,526 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail '20250710_103250_11_Inserimento nuova richiesta Eatinerari - Pacchetti di esperienze targati Eataly.eml': ORA-01401: inserted value too large for column
2025-07-16 14:16:00,016 - ERROR - 
❌ File con errore logico (doppio o altro): DEBUG - VOUCHER SISAL DIAMOND.msg
2025-07-16 14:16:02,701 - ERROR - 
❌ File con errore logico (doppio o altro): DEBUG - VOUCHER WELFARE BIT - ORDINE N.2264602.msg
2025-07-16 14:16:04,937 - ERROR - 
❌ File con errore logico (doppio o altro): DEBUG - VOUCHER WILLIS WELFARE - ORDINE N.2250481.msg
2025-07-16 14:16:07,226 - ERROR - 
❌ File con errore logico (doppio o altro): Inserimento nuova richiesta Eatinerari - A tavola con Eataly.msg
2025-07-16 14:16:09,820 - ERROR - 
❌ File con errore logico (doppio o altro): Inserimento nuova richiesta Eatinerari - Pacchetti di esperienze targati Eataly.msg
2025-07-16 14:16:12,502 - ERROR - 
❌ File con errore logico (doppio o altro): Notifica ordine 00167525.msg
2025-07-16 14:16:14,889 - ERROR - 
❌ File con errore logico (doppio o altro): Notifica ordine 00168844.msg
2025-07-16 14:16:19,380 - ERROR - 
❌ File con errore logico (doppio o altro): Notifica ordine 00194151.msg
2025-07-16 14:16:21,912 - ERROR - 
❌ File con errore logico (doppio o altro): PORTALE WELFARE PELLEGRINI CONFERMA INSERIMENTO RICHIESTA.msg
2025-07-16 14:16:25,020 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto ALESSIA MARIOTTI # SKY25043001590064.msg
2025-07-16 14:16:27,306 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto ALESSIO ARRIGO # SKY25052401704582.msg
2025-07-16 14:16:29,849 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto ALESSIO ARRIGO # SKY25052701720736.msg
2025-07-16 14:16:32,352 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto DOROTHY MAFFIA # SKY25061901874877.msg
2025-07-16 14:16:35,041 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto Giacomo Cavallo # SKY25050701621176.msg
2025-07-16 14:16:37,545 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto GIOVANNI GIORDANI # SKY25061101818808.msg
2025-07-16 14:16:41,110 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-16 14:33:18,998 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail '20250710_103250_11_Inserimento nuova richiesta Eatinerari - Pacchetti di esperienze targati Eataly.eml': ORA-01401: inserted value too large for column
2025-07-16 14:33:19,001 - ERROR - 
❌ File con errore logico (doppio o altro): 20250710_103250_11_Inserimento nuova richiesta Eatinerari - Pacchetti di esperienze targati Eataly.eml
2025-07-16 14:33:21,707 - ERROR - 
❌ File con errore logico (doppio o altro): DEBUG - VOUCHER SISAL DIAMOND.msg
2025-07-16 14:33:24,860 - ERROR - 
❌ File con errore logico (doppio o altro): DEBUG - VOUCHER WELFARE BIT - ORDINE N.2264602.msg
2025-07-16 14:33:27,372 - ERROR - 
❌ File con errore logico (doppio o altro): DEBUG - VOUCHER WILLIS WELFARE - ORDINE N.2250481.msg
2025-07-16 14:33:29,869 - ERROR - 
❌ File con errore logico (doppio o altro): Inserimento nuova richiesta Eatinerari - A tavola con Eataly.msg
2025-07-16 14:33:32,280 - ERROR - 
❌ File con errore logico (doppio o altro): Inserimento nuova richiesta Eatinerari - Pacchetti di esperienze targati Eataly.msg
2025-07-16 14:33:35,250 - ERROR - 
❌ File con errore logico (doppio o altro): Notifica ordine 00167525.msg
2025-07-16 14:33:37,824 - ERROR - 
❌ File con errore logico (doppio o altro): Notifica ordine 00168844.msg
2025-07-16 14:33:40,840 - ERROR - 
❌ File con errore logico (doppio o altro): Notifica ordine 00194151.msg
2025-07-16 14:33:43,454 - ERROR - 
❌ File con errore logico (doppio o altro): PORTALE WELFARE PELLEGRINI CONFERMA INSERIMENTO RICHIESTA.msg
2025-07-16 14:33:46,350 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto ALESSIA MARIOTTI # SKY25043001590064.msg
2025-07-16 14:33:48,955 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto ALESSIO ARRIGO # SKY25052401704582.msg
2025-07-16 14:33:51,963 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto ALESSIO ARRIGO # SKY25052701720736.msg
2025-07-16 14:33:54,493 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto DOROTHY MAFFIA # SKY25061901874877.msg
2025-07-16 14:33:57,555 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto Giacomo Cavallo # SKY25050701621176.msg
2025-07-16 14:34:00,105 - ERROR - 
❌ File con errore logico (doppio o altro): Richiesta di contatto GIOVANNI GIORDANI # SKY25061101818808.msg
2025-07-16 14:34:03,610 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-16 14:34:03,610 - ERROR - 
❌ File con errore logico (doppio o altro): test1.eml
2025-07-17 15:24:31,501 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-17 15:24:31,508 - ERROR - 
❌ File con errore logico (doppio o altro): test1.eml
2025-07-17 15:33:46,535 - ERROR - 
❌ File con errore logico (doppio o altro): DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg
2025-07-17 15:33:50,599 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-17 15:33:50,607 - ERROR - 
❌ File con errore logico (doppio o altro): test1.eml
2025-07-17 15:40:22,757 - ERROR - 
❌ File con errore logico (doppio o altro): DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg
2025-07-17 15:40:26,687 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-17 15:40:26,696 - ERROR - 
❌ File con errore logico (doppio o altro): test1.eml
2025-07-17 15:40:38,769 - ERROR - 
❌ File con errore logico (doppio o altro): DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg
2025-07-17 15:40:42,742 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-17 15:40:42,750 - ERROR - 
❌ File con errore logico (doppio o altro): test1.eml
2025-07-17 15:41:22,488 - ERROR - 
❌ File duplicato: DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg
2025-07-17 15:41:26,489 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-17 15:41:26,497 - ERROR - 
❌ File con errore sconosciuto: test1.eml
2025-07-17 15:53:21,271 - ERROR - Errore nello spostamento del file DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-17 15:53:25,029 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-17 15:53:25,038 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-17 15:53:25,038 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = N
2025-07-17 15:56:34,212 - ERROR - Errore nello spostamento del file DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-17 15:56:38,666 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-17 15:56:38,675 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-17 15:56:38,675 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = N
2025-07-17 16:03:43,650 - ERROR - Errore nello spostamento del file DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-17 16:03:47,707 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-17 16:03:47,714 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-17 16:03:47,714 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-17 16:04:52,586 - ERROR - Errore nello spostamento del file DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-17 16:04:56,125 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-17 16:04:56,133 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-17 16:04:56,133 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-17 16:09:06,449 - ERROR - Errore nello spostamento del file DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-17 16:09:10,283 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-17 16:09:10,299 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-17 16:09:10,299 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-17 16:14:05,277 - ERROR - 
❌ Il percorso specificato non è una cartella valida!
2025-07-17 16:17:22,645 - ERROR - Errore nello spostamento del file DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-17 16:17:26,611 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-17 16:17:26,625 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-17 16:17:26,625 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-21 09:57:12,986 - ERROR - 
❌ Errore nella verifica della tabella Oracle: ORA-12545: Connect failed because target host or object does not exist
2025-07-24 09:38:00,917 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 09:38:03,416 - ERROR - Errore nello spostamento del file 20250722_165954_5_Fwd_ Prova Prima.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 09:38:06,039 - ERROR - Errore nello spostamento del file DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 09:38:09,988 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-24 09:38:09,993 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-24 09:38:09,993 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-24 09:42:37,943 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 09:42:41,923 - ERROR - Errore nello spostamento del file 20250722_165954_5_Fwd_ Prova Prima.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 09:42:44,823 - ERROR - Errore nello spostamento del file DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 09:42:48,584 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-24 09:42:48,603 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-24 09:42:48,603 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-24 10:00:12,958 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-24 10:00:12,978 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-24 10:00:12,978 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-24 11:48:16,995 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 11:48:32,963 - ERROR - Errore nello spostamento del file DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 11:48:37,806 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-24 11:48:37,813 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-24 11:48:37,815 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-24 12:10:17,146 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:10:20,066 - ERROR - Errore nello spostamento del file 20250722_165954_5_Fwd_ Prova Prima.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:10:22,573 - ERROR - Errore nello spostamento del file DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:10:26,443 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-24 12:10:26,451 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-24 12:10:26,451 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-24 12:11:53,905 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:11:56,722 - ERROR - Errore nello spostamento del file DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:12:00,530 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-24 12:12:00,530 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-24 12:12:00,530 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-24 12:14:39,900 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:14:42,745 - ERROR - Errore nello spostamento del file DEBUG - VOUCHER WELFARE BIT - ORDINE N.2317452.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:14:47,202 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-24 12:14:47,213 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-24 12:14:47,213 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-24 12:16:29,032 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:16:58,100 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:17:00,624 - ERROR - Errore nello spostamento del file 20250722_165954_5_Fwd_ Prova Prima.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:18:35,592 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:18:38,087 - ERROR - Errore nello spostamento del file 20250722_165954_5_Fwd_ Prova Prima.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:18:40,947 - ERROR - Errore nello spostamento del file DEBUG - VOUCHER WILLIS WELFARE - ORDINE N.2250481.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:46:39,675 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:46:53,182 - ERROR - Errore nello spostamento del file 20250722_165954_5_Fwd_ Prova Prima.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:46:56,155 - ERROR - Errore nello spostamento del file DEBUG - VOUCHER WILLIS WELFARE - ORDINE N.2250481.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:46:59,719 - ERROR - Errore nello spostamento del file email_con_codice_barre.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:47:02,190 - ERROR - Errore nello spostamento del file email_senza_codice_barre.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 12:47:05,042 - ERROR - Errore nello spostamento del file Notifica ordine 00167525.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:13:19,362 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:13:22,035 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:19:31,055 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:19:39,267 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:21:56,839 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:22:15,293 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:22:47,042 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:22:49,755 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:33:24,672 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:33:27,387 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:34:57,492 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:35:00,122 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:38:39,789 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:38:42,203 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:41:29,779 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:41:32,561 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:50:05,536 - ERROR - 
❌ Errore nell'elaborazione del file C:\PYTHON\Email_Analyzer_v1.1.3\Mail Fittizie\20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml: Invalid format specifier ' "data della mail in formato dd/MM/YYYY",
    "numero_ordine": "numero dell'ordine o numero prenotazione o numero richiesta", 
    "nome": "nome del cliente",
    "cognome": "cognome del cliente",
    "codice_fiscale": "codice fiscale",
    "telefono": "numero di telefono",
    "articolo": "nome dell'articolo o SKU",
    "prezzo": "prezzo totale (virgola come separatore decimale)",
    "quantita": "quantità",
    "email": "email del cliente",
    "mittente": "mittente della mail",
    "codice_barre": "codice a barre dell'articolo"
' for object of type 'str'
2025-07-24 14:50:05,538 - ERROR - Traceback (most recent call last):
  File "C:\PYTHON\Email_Analyzer_v1.1.4\components\processing.py", line 25, in elabora_file
    prompt = genera_prompt(dati["body"] if isinstance(dati["body"], str) else '', dati["date"], dati["sender"], codici_barre)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\PYTHON\Email_Analyzer_v1.1.4\components\ai_utils.py", line 29, in genera_prompt
    {
ValueError: Invalid format specifier ' "data della mail in formato dd/MM/YYYY",
    "numero_ordine": "numero dell'ordine o numero prenotazione o numero richiesta", 
    "nome": "nome del cliente",
    "cognome": "cognome del cliente",
    "codice_fiscale": "codice fiscale",
    "telefono": "numero di telefono",
    "articolo": "nome dell'articolo o SKU",
    "prezzo": "prezzo totale (virgola come separatore decimale)",
    "quantita": "quantità",
    "email": "email del cliente",
    "mittente": "mittente della mail",
    "codice_barre": "codice a barre dell'articolo"
' for object of type 'str'

2025-07-24 14:50:05,539 - ERROR - 
❌ Errore nell'elaborazione del file: 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml
2025-07-24 14:50:06,542 - ERROR - 
❌ Errore nell'elaborazione del file C:\PYTHON\Email_Analyzer_v1.1.3\Mail Fittizie\20250724_130300_2_I_ PORTALE PRIMA.eml: Invalid format specifier ' "data della mail in formato dd/MM/YYYY",
    "numero_ordine": "numero dell'ordine o numero prenotazione o numero richiesta", 
    "nome": "nome del cliente",
    "cognome": "cognome del cliente",
    "codice_fiscale": "codice fiscale",
    "telefono": "numero di telefono",
    "articolo": "nome dell'articolo o SKU",
    "prezzo": "prezzo totale (virgola come separatore decimale)",
    "quantita": "quantità",
    "email": "email del cliente",
    "mittente": "mittente della mail",
    "codice_barre": "codice a barre dell'articolo"
' for object of type 'str'
2025-07-24 14:50:06,542 - ERROR - Traceback (most recent call last):
  File "C:\PYTHON\Email_Analyzer_v1.1.4\components\processing.py", line 25, in elabora_file
    prompt = genera_prompt(dati["body"] if isinstance(dati["body"], str) else '', dati["date"], dati["sender"], codici_barre)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\PYTHON\Email_Analyzer_v1.1.4\components\ai_utils.py", line 29, in genera_prompt
    {
ValueError: Invalid format specifier ' "data della mail in formato dd/MM/YYYY",
    "numero_ordine": "numero dell'ordine o numero prenotazione o numero richiesta", 
    "nome": "nome del cliente",
    "cognome": "cognome del cliente",
    "codice_fiscale": "codice fiscale",
    "telefono": "numero di telefono",
    "articolo": "nome dell'articolo o SKU",
    "prezzo": "prezzo totale (virgola come separatore decimale)",
    "quantita": "quantità",
    "email": "email del cliente",
    "mittente": "mittente della mail",
    "codice_barre": "codice a barre dell'articolo"
' for object of type 'str'

2025-07-24 14:50:06,542 - ERROR - 
❌ Errore nell'elaborazione del file: 20250724_130300_2_I_ PORTALE PRIMA.eml
2025-07-24 14:51:42,999 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:51:45,587 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:52:10,532 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 14:52:12,819 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:05:55,779 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:05:58,318 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:06:07,602 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:06:10,194 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:13:06,627 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:13:09,627 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:13:36,165 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:13:39,317 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:14:20,691 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:14:22,998 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:15:37,063 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:15:39,907 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:30:43,874 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:30:46,751 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:34:29,816 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:34:35,327 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:36:14,683 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:36:17,460 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:37:51,500 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:37:53,994 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:44:08,476 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:44:11,030 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:45:10,759 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:45:13,231 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:45:31,860 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:45:34,293 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:52:14,703 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:52:17,509 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:52:46,616 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:52:50,133 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:52:57,904 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:53:00,464 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:53:08,186 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:53:10,939 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:53:46,313 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:53:49,321 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:55:12,020 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:55:14,733 - ERROR - Errore nello spostamento del file 20250722_165954_5_Fwd_ Prova Prima.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:55:17,339 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:55:19,862 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:58:16,642 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:58:19,206 - ERROR - Errore nello spostamento del file 20250722_165954_5_Fwd_ Prova Prima.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:58:21,915 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 15:58:24,658 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:00:30,817 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:00:33,981 - ERROR - Errore nello spostamento del file 20250722_165954_5_Fwd_ Prova Prima.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:00:41,384 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:00:44,826 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:01:55,979 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:01:58,615 - ERROR - Errore nello spostamento del file 20250722_165954_5_Fwd_ Prova Prima.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:02:01,257 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:02:03,806 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:04:06,255 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:04:09,068 - ERROR - Errore nello spostamento del file 20250722_165954_5_Fwd_ Prova Prima.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:04:11,623 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:04:14,284 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:05:51,403 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:05:54,166 - ERROR - Errore nello spostamento del file 20250722_165954_5_Fwd_ Prova Prima.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:05:57,043 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:06:00,118 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:06:07,996 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:06:11,143 - ERROR - Errore nello spostamento del file 20250722_165954_5_Fwd_ Prova Prima.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:06:14,346 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:06:17,052 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:10:19,350 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:10:22,131 - ERROR - Errore nello spostamento del file 20250722_165954_5_Fwd_ Prova Prima.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:10:24,939 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:10:27,455 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:16:04,416 - ERROR - Errore nello spostamento del file 20250722_165953_1_Fwd_ prova dopo.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:16:08,900 - ERROR - Errore nello spostamento del file 20250722_165954_5_Fwd_ Prova Prima.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:16:12,494 - ERROR - Errore nello spostamento del file 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 16:16:15,627 - ERROR - Errore nello spostamento del file 20250724_130300_2_I_ PORTALE PRIMA.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:03:12,491 - ERROR - Errore nello spostamento del file 20250710_103250_11_Inserimento nuova richiesta Eatinerari - Pacchetti di esperienze targati Eataly.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:03:15,946 - ERROR - Errore nello spostamento del file DEBUG - VOUCHER SISAL DIAMOND.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:03:18,896 - ERROR - Errore nello spostamento del file DEBUG - VOUCHER WELFARE BIT - ORDINE N.2264602.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:03:22,383 - ERROR - Errore nello spostamento del file DEBUG - VOUCHER WILLIS WELFARE - ORDINE N.2250481.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:03:32,650 - ERROR - Errore nello spostamento del file Inserimento nuova richiesta Eatinerari - A tavola con Eataly.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:03:35,709 - ERROR - Errore nello spostamento del file Inserimento nuova richiesta Eatinerari - Pacchetti di esperienze targati Eataly.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:03:38,384 - ERROR - Errore nello spostamento del file Notifica ordine 00167525.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:03:46,489 - ERROR - Errore nello spostamento del file Notifica ordine 00168844.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:03:49,265 - ERROR - Errore nello spostamento del file Notifica ordine 00194151.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:03:51,907 - ERROR - Errore nello spostamento del file PORTALE WELFARE PELLEGRINI CONFERMA INSERIMENTO RICHIESTA.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:03:54,609 - ERROR - Errore nello spostamento del file Richiesta di contatto ALESSIA MARIOTTI # SKY25043001590064.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:03:57,471 - ERROR - Errore nello spostamento del file Richiesta di contatto ALESSIO ARRIGO # SKY25052401704582.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:04:00,140 - ERROR - Errore nello spostamento del file Richiesta di contatto ALESSIO ARRIGO # SKY25052701720736.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:04:03,096 - ERROR - Errore nello spostamento del file Richiesta di contatto DOROTHY MAFFIA # SKY25061901874877.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:04:05,574 - ERROR - Errore nello spostamento del file Richiesta di contatto Giacomo Cavallo # SKY25050701621176.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:04:08,132 - ERROR - Errore nello spostamento del file Richiesta di contatto GIOVANNI GIORDANI # SKY25061101818808.msg nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:04:12,286 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-24 17:04:12,286 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-24 17:04:12,301 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-24 17:04:14,692 - ERROR - Errore nello spostamento del file testdebub.eml nella cartella storico: [WinError 5] Accesso negato: 'C:\\Users\\batti'
2025-07-24 17:40:02,822 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-24 17:40:02,858 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-24 17:40:02,860 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-24 17:42:45,890 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-24 17:42:45,902 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-24 17:42:45,902 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-25 09:51:54,805 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-25 09:51:54,811 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-25 09:51:54,811 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-25 09:59:10,295 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-25 09:59:10,295 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-25 09:59:10,295 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-25 10:07:48,351 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-25 10:07:48,367 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-25 10:07:48,367 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-25 10:10:25,051 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': ORA-01401: inserted value too large for column
2025-07-25 10:10:25,062 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-25 10:10:25,064 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-28 12:54:56,744 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail '20250725_124008_2_PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml': 'numero_ordine'
2025-07-28 12:54:56,761 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-28 12:54:56,761 - ERROR - 
❌ File con errore sconosciuto: 20250725_124008_2_PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml, Stato = O
2025-07-28 12:54:57,769 - ERROR - 
❌ Errore nell'elaborazione del file: MailMalevola.eml
2025-07-28 12:56:33,389 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail '20250722_165953_1_Fwd_ prova dopo.eml': 'numero_ordine'
2025-07-28 12:56:33,389 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-28 12:56:33,389 - ERROR - 
❌ File con errore sconosciuto: 20250722_165953_1_Fwd_ prova dopo.eml, Stato = O
2025-07-28 12:56:37,313 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail '20250722_165954_5_Fwd_ Prova Prima.eml': 'numero_ordine'
2025-07-28 12:56:37,318 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-28 12:56:37,318 - ERROR - 
❌ File con errore sconosciuto: 20250722_165954_5_Fwd_ Prova Prima.eml, Stato = O
2025-07-28 12:56:40,831 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail '20250724_125107_2_DEBUG - VOUCHER MY SISAL.eml': 'numero_ordine'
2025-07-28 12:56:40,847 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-28 12:56:40,847 - ERROR - 
❌ File con errore sconosciuto: 20250724_125107_2_DEBUG - VOUCHER MY SISAL.eml, Stato = O
2025-07-28 12:56:44,429 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail '20250724_125107_3_[ESTERNO] Inserimento nuova richiesta Eatinerari - A tavola con Eataly.eml': 'numero_ordine'
2025-07-28 12:56:44,429 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-28 12:56:44,429 - ERROR - 
❌ File con errore sconosciuto: 20250724_125107_3_[ESTERNO] Inserimento nuova richiesta Eatinerari - A tavola con Eataly.eml, Stato = O
2025-07-28 12:56:48,202 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail '20250724_125107_6_PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml': 'numero_ordine'
2025-07-28 12:56:48,210 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-28 12:56:48,210 - ERROR - 
❌ File con errore sconosciuto: 20250724_125107_6_PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml, Stato = O
2025-07-28 12:56:51,785 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail '20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml': 'numero_ordine'
2025-07-28 12:56:51,793 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-28 12:56:51,793 - ERROR - 
❌ File con errore sconosciuto: 20250724_130300_1_I_ PORTALE WELFARE PELLEGRINI_ CONFERMA INSERIMENTO RICHIESTA.eml, Stato = O
2025-07-28 12:56:55,196 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'DEBUG - VOUCHER WILLIS WELFARE - ORDINE N.2250481.msg': 'numero_ordine'
2025-07-28 12:56:55,212 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-28 12:56:55,212 - ERROR - 
❌ File con errore sconosciuto: DEBUG - VOUCHER WILLIS WELFARE - ORDINE N.2250481.msg, Stato = O
2025-07-28 12:56:58,778 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'Notifica ordine 00194151.msg': 'numero_ordine'
2025-07-28 12:56:58,794 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-28 12:56:58,794 - ERROR - 
❌ File con errore sconosciuto: Notifica ordine 00194151.msg, Stato = O
2025-07-28 12:57:02,370 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'Richiesta di contatto GIOVANNI GIORDANI # SKY25061101818808.msg': 'numero_ordine'
2025-07-28 12:57:02,370 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-28 12:57:02,370 - ERROR - 
❌ File con errore sconosciuto: Richiesta di contatto GIOVANNI GIORDANI # SKY25061101818808.msg, Stato = O
2025-07-28 12:57:06,395 - ERROR - 
❌ Errore nell'inserimento dati in Oracle per la mail 'test1.eml': 'numero_ordine'
2025-07-28 12:57:06,410 - ERROR - 
❌ Nessun record inserito nel database Oracle.
2025-07-28 12:57:06,410 - ERROR - 
❌ File con errore sconosciuto: test1.eml, Stato = O
2025-07-28 12:59:46,158 - ERROR - 
❌ Errore nell'elaborazione del file: MailMalevola.eml
2025-07-28 13:02:17,263 - ERROR - 
❌ Errore nell'elaborazione del file: MailMalevola.eml
2025-07-28 14:15:53,921 - ERROR - 
❌ Il percorso specificato non è una cartella valida!
